doc/examples/onnx_importer.md

```diff
@@ -0,0 +1,88 @@
+# ONNX importer usage examples
+
+This sample demonstrates how to use ONNX importer API.
+The API allows to create nGraph Function from ONNX importer model.
+
+All functions forming ONNX importer API are in [onnx.hpp][onnx_header] header file.
+The available functions can be divided into two categories from user's point of view.
+The first group contains helpers' functions used to check which ONNX op are supported in current version of ONNX importer.
+The second group contains function used to read ONNX models from stream or file. Their result is nGraph function which can be calculated on Inference Engine.
+
+## How to check which ONNX ops are supported
+
+In order to list all supported ONNX ops in a specific version and domain `get_supported_operators` function should be used.
+The code listing below shows how to do it.
+```
+const std::int64_t version = 12;
+const std::string domain = "ai.onnx";
+const std::set<std::string> supported_ops = ngraph::onnx_import::get_supported_operators(version, domain);
+
+for(const auto& op : supported_ops)
+{
+    std::cout << op << std::endl;
+}
+```
+Fragment of output:
+```
+Abs
+Acos
+...
+Xor
+```
+
+In order to check if ONNX op in specific version and domain is supported by current version of ONNX importer `is_operator_supported` function should be used.
```

Comment:
To determine whether an ONNX op in a specific version and domain is supported by the importer

Reply:
`To determine whether an ONNX op in a specific version and domain is supported by the importer, use the function` version used